### PR TITLE
@Valid 테스트 & DTO

### PR DESCRIPTION
**Pull request by Jungwon423**

### CODE REVIEW

**Comment by oreo1001** [comment]
LGTM

**Review by oreo1001 — COMMENTED** [review]
submit

File: src/main/java/HotDeal/HotDeal/Controller/HomeController.java [hunk 1]
```diff
@@ -17,15 +22,7 @@ public class HomeController {
...
-    @GetMapping("test2")
-    public String test2() { return "깃헙 텍스트"; }
-
-    @PostMapping("insert")
-    public String insertTest(@RequestBody Map<String, Object> map) {
-        return homeService.insertTest((String) map.get("name"));
+    public ResponseEntity<TestDto> test(@Valid @RequestBody TestDto testDto) {
+        return ResponseEntity.status(HttpStatus.OK).body(testDto);
     }
 }
```
Comment (oreo1001) [inline src/main/java/HotDeal/HotDeal/Controller/HomeController.java:28]: dsgspojpsdgspdgpsg

**Review by oreo1001 — COMMENTED** [review]
Lgtb

**Review by meque98 — COMMENTED** [review]
GLHF

**Review by oreo1001 — APPROVED** [review]
final

**Review by meque98 — APPROVED** [review]

**Pull request merged**